python/tvm/driver/tvmc/transform.py

```diff
@@ -123,7 +125,29 @@ def convert_graph_layout(mod, desired_layout, ops=None):
     if ops is None:
         ops = ["nn.conv2d", "nn.conv2d_transpose", "qnn.conv2d"]
 
-    desired_layouts = {op: [desired_layout, "default"] for op in ops}
+    if not isinstance(desired_layouts, list):
+        # For backwards compatibility
+        assert isinstance(desired_layouts, str)
+        desired_layouts = [desired_layouts]
+
+    assert len(desired_layouts) > 0
```

Comment:
Nit: This check seems redundant to me, I think if no layouts are provided the convert layout pass won't do anything - would it be okay to check?

Reply:
It is in fact redundant, especially since I use `nargs='+'`.

I will drop it when fixing the CI bugs